__tests__/data/salesforce_proxy_output.json

```diff
@@ -0,0 +1,63 @@
+[
+  {
+    "output": {
+      "status": 204,
+      "message": "Request for destination: salesforce Processed Successfully",
+      "destinationResponse": {
+        "response": "",
+        "status": 204
+      }
+    }
+  },
+  {
+    "output": {
+      "status": 400,
+      "message": "Salesforce Request Failed: 500 due to , Aborted.",
+      "destinationResponse": {
+        "response": "",
+        "status": 500
+      },
+      "statTags": {
+        "destType": "SALESFORCE",
+        "errorAt": "proxy",
+        "scope": "api",
+        "stage": "responseTransform",
+        "meta": "abortable"
+      }
+    }
+  },
+  {
+    "output": {
+      "status": 400,
+      "message": "Salesforce Request Failed: 500 due to , Aborted.",
+      "destinationResponse": {
+        "response": "",
+        "status": 500
+      },
+      "statTags": {
+        "destType": "SALESFORCE",
+        "errorAt": "proxy",
+        "scope": "api",
+        "stage": "responseTransform",
+        "meta": "abortable"
+      }
+    }
+  },
+  {
+    "output": {
+      "status": 400,
+      "message": "Salesforce Request Failed: 500 due to , Aborted.",
+      "destinationResponse": {
+        "response": "",
+        "status": 500
```

Comment:
If salesforce responds with `5xx` we should retry the request("status" should be `500`) right ?

Reply:
Seems like `500` is ok to be aborted in Salesforce case. Can we add test-case for `503` ?

Reply:
Also, are we aborting for `REQUEST_LIMIT_EXCEEDED` as well ?

Reply:
yes, added that, and updated the testcase.